### ISSUE TITLE
Failing to import type

### ISSUE THREAD

**Issue by andresespinosapc** [ardatan/graphql-import#291]
I have troubles trying to import a schema generated by postgraphile.

**Attempt 1**
I first tried importing a specific type like this: `# import ProdAttrsConnection from './generated/postgraphile.graphql'`

After writing the import line, I get this error with different types: `Unknown type "ProductToAttrsOrderBy".` The types that throw the error are not even dependencies of the one I imported, so I think it is a schema validation error. In the case of the error I mentioned, `ProductToAttrsOrderBy` is an enum and it is declared in the schema.

**Attempt 2**
Then I tried importing the whole schema like this: `# import * from ./generated/postgraphile.graphql`

In this case, I don't get an error by just writing the import line, but when I use any imported type I get this error with multiple types: `Interface field Node.nodeId expected but ProdItemAttr does not provide it.`

Here are the types corresponding to the error I mentioned:

```graphql
interface Node {
  nodeId: ID!
}

type ProdItemAttr implements Node {
  nodeId: ID!
  # ...otherStuff
}
```

**Comment by jjangga0214** [comment]
For Attempt 2, there's already an issue, #315

**Comment by ardatan** [comment]
I think that issue is fixed in v1.0.0! Feel free to open a new one if it still persists.
